### PR TITLE
Refator #424

### PR DESCRIPTION
**Pull request by fisker**

### CODE REVIEW

**Comment by fisker** [comment]
Still some DRY work to do

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
 		"xo": "^0.25.3"
 	},
 	"peerDependencies": {
-		"eslint": ">=6.3.0"
+		"eslint": ">=6.6.0"
```
Comment (voxpelli) [inline package.json]: Are the update of dependencies intentional?

Comment (fisker) [reply package.json]: yes, code base already updated https://github.com/sindresorhus/eslint-plugin-unicorn/pull/425

**Comment by fisker** [comment]
You don't have to use my code, just send here to let you know what will I do, what do you think?

**Comment by voxpelli** [comment]
@fisker I'm not sure the extraction into a function helps with the readability, I may be biased as I've just written my solution, but I leans towards it being more readable

**Comment by fisker** [comment]
the purpose move to a  function is we can use early-return

**Comment by voxpelli** [comment]
@fisker Yeah, I'm not just sure it makes things more readable and simpler. The if-statement is very compact and clear regarding which checks are done and in which order

**Pull request closed without merging**